### PR TITLE
fix Kubernetes Install guide for Cilium 

### PR DESCRIPTION
**Pull request by wyyxd2017**
Kubernetes Install guide for Cilium

### CODE REVIEW

**Comment by remyleone** [comment]
@wyyxd2017 could you make a PR per localization? Localization teams don't have authority to merge PR that are cross languages.

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/canal.yaml
 {{% /tab %}}
 
 {{% tab name="Cilium" %}}
-For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/kubernetes/).
+For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-managed/).
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:322]: thank you for the PR, but i think the old page lets the user navigate to a section of the Cilium docs they want, while the new proposed link sends them to managed k8s, which they might not be using.

a potentially better link might be:
https://docs.cilium.io/en/stable/gettingstarted/#installation

/cc @aanm 
/hold 
/sig cluster-lifecycle

Comment (aanm) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:322]: @neolit123 thanks for the heads up, given this is kubeadm, the link should be:

https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/

File: content/fr/docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -326,7 +326,7 @@ kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/canal.yaml
 
 {{% tab name="Cilium" %}}
 Pour plus d'informations sur l'utilisation de Cilium avec Kubernetes, voir
-[Guide d'installation de Kubernetes pour Cilium](https://docs.cilium.io/en/stable/kubernetes/).
+[Guide d'installation de Kubernetes pour Cilium](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-managed/).
```
Comment (aanm) [inline content/fr/docs/setup/independent/create-cluster-kubeadm.md:329]: https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/

File: content/ja/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 3]
```diff
@@ -309,7 +309,7 @@ kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/canal.yaml
 {{% /tab %}}
 
 {{% tab name="Cilium" %}}
-For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/kubernetes/).
+For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-managed/).
```
Comment (aanm) [inline content/ja/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:312]: https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/

File: content/zh/docs/concepts/configuration/manage-compute-resources-container.md [hunk 4]
```diff
@@ -387,7 +387,7 @@ with namespaces, it can prevent one team from hogging all the resources.
 
 Pod 可用的资源量小于节点容量，因为系统守护程序使用一部分可用资源。
 [NodeStatus](/docs/resources-reference/{{< param "version" >}}/#nodestatus-v1-core) 的 `allocatable` 字段给出了可用于 Pod 的资源量。
-有关更多信息，请参阅 [节点可分配资源](https://git.k8s.io/community/contributors/design-proposals/node-allocatable.md)。
+有关更多信息，请参阅 [节点可分配资源](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/node-allocatable.md)。
```
Comment (aanm) [inline content/zh/docs/concepts/configuration/manage-compute-resources-container.md:390]: This is not related with Cilium AFAIK.

**Comment by kbhawkey** [comment]
👋 @wyyxd2017 . 
Would you have time to create changes for only one language, address the reviewer comments, and rebase?
With these changes, this pull request can move forward. Thanks!

**Comment by neolit123** [comment]
> content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

please note that this page no longer includes CNI quick start guides.

**Comment by zacharysarah** [comment]
@wyyxd2017 Feel free to `/reopen` when you're ready to rebase and continue.

/close
